src/main/java/org/takes/misc/Href.java

```diff
@@ -59,12 +59,13 @@
     /**
      * Ctor.
      * @param txt Text of the link
+     *  Href use undecoded query for URL creation
```

Comment:
It's a strange place for a comment. Try to build project site with `mvn clean site -Psite` and open `./target/site/apidocs-1.0-SNAPSHOT/index.html`. Then, take a look how this particular method is documented, you will see the problem